src/hb-ot-shaper-indic.cc

```diff
@@ -39,6 +39,66 @@
  */
 
 
+#define I_Check(C) static_assert (OT_##C == I_Cat(C), "")
```

Comment:
My current thinking is this, but I'll give it another try:
```
/* Note: This enum is duplicated the machine machine.rl files.
 * We can avoid that by defining this enum in terms of those in the
 * indic-table.cc file, but I like this enum duplicated here, because
 * this gives us a unified view of all the numbers.
```

Reply:
I still don’t like duplication, but with all the comments and checks I admit the risk is low.